File: 0001 Code Chef/0014 Add two numbers/implementation/0008 python3/add_two_number.py
if __name__ == "__main__":
    string_input = input()
    test = int(string_input)
    while(test > 0):
        string_input = input()
        string_array = string_input.split(" ")
        x = int(string_array[0])
        y = int(string_array[1])
        print(x+y)
        test = test - 1

File: 0001 Code Chef/0014 Add two numbers/implementation/0005 kotlin/AddTwoNumber.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var test:Int = stringInput.toInt();
        while(test > 0){
            var userInput:String? = readLine();
            if(userInput != null){
                var stringArray:Array<String> = userInput.split(" ").toTypedArray();
                if(stringArray.size  == 2){
                    var x:Int = stringArray[0].toInt();
                    var y:Int = stringArray[1].toInt();
                    println(x+y);
                }
            }

            test = test - 1;
        }
    }else{
    println("Invalid input");
   }
}